Configuration/TCA/tx_dlf_documents.php

```diff
@@ -385,7 +385,7 @@
                 'foreign_table_where' => 'AND tx_dlf_collections.pid=###CURRENT_PID### AND tx_dlf_collections.sys_language_uid IN (-1,0) ORDER BY tx_dlf_collections.label',
                 'size' => 5,
                 'autoSizeMax' => 15,
-                'minitems' => 1,
+                'minitems' => 0,
```

Comment:
I'm not sure about this change. Of course I can live with a required field less. But are you sure, all plugins work with an empty collections field?

What's the relation with the collections field and the OAI harvesting feature?

Reply:
As far as I checked, there are no plugins requiring the collection field. The OAI plugin uses collections as OAI sets, but that's optional.

Reply:
I agree. This is the expected behaviour.

I found only the feeds plugin. New documents without collection are not listed in the RSS feed. But this should be discussed on the plugin side.